src/kili/adapters/kili_api_gateway/helpers/queries.py

```diff
@@ -35,8 +36,9 @@ class PaginatedGraphQLQuery:
     It factorizes code for executing paginated queries.
     """
 
-    def __init__(self, graphql_client: GraphQLClient):
+    def __init__(self, graphql_client: GraphQLClient, http_client: HttpClient):
         self._graphql_client = graphql_client
+        self._http_client = http_client
```

Comment:
why do we need the http client?

Reply:
okay, so we indeed don't need it currently. 
I added it because I used it and that I didn't delete it in case we need it in the future, not to change plenty of files. If you prefer to delete it I can, I don't have a strong opinion on that

Reply:
I would not keep it. I don't think we would need it in the future in this part of the code, since we just query list/dict data here. We're not supposed to preprocess/parse it